src/lookup/responseLookup.ts

```diff
@@ -46,14 +46,19 @@ const lookup: ResponseDict = {
     SPEAKER_CANNOT_BE_REJECTED:                   'Dieser Speaker kann nicht abgelehnt werden.',
 
     // talk
-    TALK_ALREADY_APPROVED: 'Der Talk ist bereits freigegeben',
-    TALK_NOT_FOUND:        'Da haben wir wohl diesen Talk verlegt. Vielleicht finden wir ihn bald wieder.',
-    DUPLICATE_TALK:        'Diesen Talk gibt es bereits',
+    TALK_ALREADY_APPROVED:                         'Der Talk ist bereits freigegeben.',
+    TALK_NOT_FOUND:                                'Da haben wir wohl diesen Talk verlegt. Vielleicht finden wir ihn bald wieder.',
+    DUPLICATE_TALK:                                'Diesen Talk gibt es bereits.',
+    'The title field is required.':                'Das Feld \'Titel\' ist erforderlich.',
+    'The description field is required.':          'Das Feld \'Beschreibung\' ist erforderlich.',
+    'The tag_ids.* field is required.':            'Das Feld \'Tags\' ist erforderlich.',
```

Comment:
Since this is a user-facing message, I think we should be a bit more user-friendly here.
```suggestion
    'The tag_ids.* field is required.':            'Wähle durch Anklicken mindestens einen Tag aus, der zu deinem Vortrag passt.',
```